docs/timeseries.md

```diff
@@ -1,4 +1,4 @@
-# Timevector [<sup><mark>experimental</mark></sup>](/docs/README.md#tag-notes)
```

Comment:
Is this stabilized? https://github.com/timescale/timescaledb-toolkit/issues/483

Reply:
Timevectors are stabilized, timevector pipelines are not